oauth2_http/java/com/google/auth/mtls/X509Provider.java

```diff
@@ -90,8 +119,10 @@ public X509Provider() {
    * @throws IOException if there is an error retrieving the certificate configuration.
    */
   public KeyStore getKeyStore() throws IOException {
-
-    WorkloadCertificateConfiguration workloadCertConfig = getWorkloadCertificateConfiguration();
+    if (loadedConfig == null) {
+      loadedConfig = getWorkloadCertificateConfiguration();
+    }
+    WorkloadCertificateConfiguration workloadCertConfig = loadedConfig;
```

Comment:
qq, do we need this local workloadCertConfig object? Can we just read from `loadedConfig`?